tools/run-clang-tidy.py

```diff
@@ -0,0 +1,388 @@
+#!/usr/bin/env python
```

Comment:
@msbaines is this custom script for Habitat or it can be `pip` installed?

Reply:
The script comes from here:

https://github.com/llvm-mirror/clang-tools-extra/blob/master/clang-tidy/tool/run-clang-tidy.py

I don't believe its available via pip install.

On Mac, you will also need to:

brew install llvm